### DISCUSSION TITLE
compile error TFormBeforeShownMessage in Delphi XE8

### DISCUSSION THREAD

**Discussion by ScottHollows** [skia4delphi/skia4delphi#253] [General]
I installed the latest version of Skia3Delphi in a Delphi XE8 install

Im getting this compile error when comping in DEBUG build  configuration.
The error does not occuring when compiloing in RELEASE build configuration

_[dcc32 Error] Skia.FMX.Graphics.pas(3435): E2003 Undeclared identifier: 'TFormBeforeShownMessage'_

The code that it is failing on is

**FILE** Skia.FMX.Graphics.pas

constructor TSkCanvasService.Create(const ACurrent: IFMXCanvasService);
begin
  inherited Create;
  FCurrent := ACurrent;
  {$IFDEF DEBUG}
    FFormBeforeShownMessageId := TMessageManager.DefaultManager.SubscribeToMessage(**TFormBeforeShownMessage**, FormBeforeShownHandler);
  {$ENDIF}
end;

P.S I did the right click >> Enable Skia and also added these USES to the forms that use Skia3Delphi to use WEBP images

USES Skia, Skia.Fmx;

What do I need to do to fix the error ?
I cant find any file that has the source code for TFormBeforeShownMessage

Environment
- Delphi XE8
- FireMonkey project
- Windows 10

![image](https://github.com/skia4delphi/skia4delphi/assets/4010210/7d8eb4ab-17c2-47be-b64c-e4b151f12834)

**Comment by viniciusfbb** [comment]
This issue is fixed on version [6.0.0-beta3](https://github.com/skia4delphi/skia4delphi/releases)

**Comment by ScottHollows** [comment]
I will install that. Thank you
